README.md

```diff
@@ -71,17 +71,19 @@ If you are using Browserify to build your project, you will need to add the foll
 
 ### webpack
 
-If you are using webpack, you will need the following loaders:
+If you are using webpack, you will need to install loaders (`$ npm install --save brfs ejs-compiled-loader json-loader packageify transform-loader`) and then use them in your `webpack.config.js` file:
 
 ```js
-[
-  {
-    test: /node_modules\/auth0-lock\/.*\.js$/,
-    loaders: 'transform/cacheable?brfs!transform/cacheable?packageify'
```

Comment:
Typo! Either use `loaders` (with an "s") with the array syntax, or use singular `loaders` with the string syntax. This gets me every time with webpack, lol.